content/en/docs/v3.5/install.md

```diff
@@ -86,14 +86,122 @@ $ brew update
 $ brew install etcd
 ```
 
-3. Verify install 
+3. Verify install
 ```sh
 $ etcd --version
 ```
 
 ## Linux
 
-TBD
+Although it is possible to install etcd through major distribution's official repositories and package managers, the versions published are significantly outdated. So, installing this way is strongly discouraged.
+
+The following package managers are community maintained and have the most up-to-date version:
+
+### Homebrew on Linux
+
+[Homebrew can run on Linux], and can provide recent software versions.
+
+* Prerequisites
+  * Update Homebrew:
+    ```sh
```

Comment:
As above, let's try tidy up markdownlint at the same time.

> MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank linesmarkdownlint[MD031](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md031.md)

Reply:
Done.